lib/rules/jsx-sort-default-props.js

```diff
@@ -87,11 +87,12 @@ module.exports = {
 
     /**
      * Find a variable by name in the current scope.
-     * @param  {string} name Name of the variable to look for.
+     * @param {ASTNode} node The node to check.
      * @returns {ASTNode|null} Return null if the variable could not be found, ASTNode otherwise.
      */
-    function findVariableByName(name) {
-      const variable = variableUtil.variablesInScope(context).find((item) => item.name === name);
+    function findVariableByName(node) {
+      const name = node.name;
+      const variable = variableUtil.variablesInScope(context, node).find((item) => item.name === name);
```

Comment:
it seems like `variableUtil.variablesInScope(context, node, node.name)` should do this logic for us. am i misreading?

Reply:
`variableUtil.variablesInScope(context, node)` returns a set of vars visible from the specified node, and then we pick the one by name, as I understand.

Reply:
right - but providing the third argument to `variablesInScope` picks it by name already, doesn't it?